TypeError in `TestAnthropicStandard.test_with_structured_output[PersonB]`: `'ModelMetaclass' object is not a mapping`

### Checked other resources

- [x] This is a bug, not a usage question.
- [x] I added a clear and descriptive title that summarizes this issue.
- [x] I used the GitHub search to find a similar question and didn't find it.
- [x] I am sure that this is a bug in LangChain rather than my code.
- [x] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).
- [x] This is not related to the langchain-community package.
- [x] I posted a self-contained, minimal, reproducible example. A maintainer can copy it and run it AS IS.

### Package (Required)

- [ ] langchain
- [ ] langchain-openai
- [x] langchain-anthropic
- [ ] langchain-classic
- [ ] langchain-core
- [ ] langchain-cli
- [ ] langchain-model-profiles
- [ ] langchain-tests
- [ ] langchain-text-splitters
- [ ] langchain-chroma
- [ ] langchain-deepseek
- [ ] langchain-exa
- [ ] langchain-fireworks
- [ ] langchain-groq
- [ ] langchain-huggingface
- [ ] langchain-mistralai
- [ ] langchain-nomic
- [ ] langchain-ollama
- [ ] langchain-perplexity
- [ ] langchain-prompty
- [ ] langchain-qdrant
- [ ] langchain-xai
- [ ] Other / not sure / general

### Example Code (Python)

```python
Running pytest on langchain-anthropic 1.1.0 while building for nixpkgs
```

### Error Message and Stack Trace (if applicable)

```shell
=================================== FAILURES ===================================
__________ TestAnthropicStandard.test_with_structured_output[PersonB] __________

self = <tests.unit_tests.test_standard.TestAnthropicStandard object at 0x7ffff2cd6030>
model = ChatAnthropic(model='claude-3-haiku-20240307', max_tokens=100, temperature=0.0, default_request_timeout=60.0, stop_sequences=[], anthropic_api_url='https://api.anthropic.com', anthropic_api_key=SecretStr('**********'), model_kwargs={})
schema = <class 'langchain_tests.unit_tests.chat_models.generate_schema_pydantic_v1_from_2.<locals>.PersonB'>

    @pytest.mark.parametrize("schema", TEST_PYDANTIC_MODELS)
    def test_with_structured_output(
        self,
        model: BaseChatModel,
        schema: Any,
    ) -> None:
        """Test `with_structured_output` method.

        Test is skipped if the `has_structured_output` property on the test class is
        False.

        ??? note "Troubleshooting"

            If this test fails, ensure that the model's `bind_tools` method
            properly handles Pydantic V2 models. `langchain_core` implements
            a utility function that will accommodate most formats: https://python.langchain.com/api_reference/core/utils/langchain_core.utils.function_calling.convert_to_openai_tool.html

            See example implementation of `with_structured_output` here: https://python.langchain.com/api_reference/_modules/langchain_openai/chat_models/base.html#BaseChatOpenAI.with_structured_output

        """
        if not self.has_structured_output:
            return

        assert model.with_structured_output(schema) is not None
        for method in ["json_schema", "function_calling", "json_mode"]:
            strict_values = [None, False, True] if method != "json_mode" else [None]
            for strict in strict_values:
>               assert model.with_structured_output(
                    schema, method=method, strict=strict
                )

/nix/store/d26gd8dv736q1fvkmcvlp3rb4gws9qnf-python3.13-langchain-tests-1.0.0/lib/python3.13/site-packages/langchain_tests/unit_tests/chat_models.py:1005:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
langchain_anthropic/chat_models.py:2397: in with_structured_output
    output_format=_convert_to_anthropic_output_format(schema),
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
langchain_anthropic/chat_models.py:2596: in _convert_to_anthropic_output_format
    json_schema = transform_schema(schema)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

json_schema = <class 'langchain_tests.unit_tests.chat_models.generate_schema_pydantic_v1_from_2.<locals>.PersonB'>

    def transform_schema(
        json_schema: type[pydantic.BaseModel] | dict[str, Any],
    ) -> dict[str, Any]:
        """
        Transforms a JSON schema to ensure it conforms to the API's expectations.

        Args:
            json_schema (Dict[str, Any]): The original JSON schema.

        Returns:
            The transformed JSON schema.

        Examples:
            >>> transform_schema(
            ...     {
            ...         "type": "integer",
            ...         "minimum": 1,
            ...         "maximum": 10,
            ...         "description": "A number",
            ...     }
            ... )
            {'type': 'integer', 'description': 'A number\n\n{minimum: 1, maximum: 10}'}
        """
        if inspect.isclass(json_schema) and issubclass(json_schema, pydantic.BaseModel):  # pyright: ignore[reportUnnecessaryIsInstance]
            json_schema = json_schema.model_json_schema()

        strict_schema: dict[str, Any] = {}
>       json_schema = {**json_schema}
                      ^^^^^^^^^^^^^^^
E       TypeError: 'ModelMetaclass' object is not a mapping

/nix/store/091j0qyazshsy6g4z5m5d0m13qsgqs7a-python3.13-anthropic-0.74.1/lib/python3.13/site-packages/anthropic/lib/_parse/_transform.py:81: TypeError
=============================== warnings summary ===============================
tests/unit_tests/middleware/test_prompt_caching.py::test_anthropic_prompt_caching_middleware_async_min_messages
  /nix/var/nix/builds/nix-build-python3.13-langchain-anthropic-1.1.0.drv-0/b/source/libs/partners/anthropic/tests/unit_tests/middleware/test_prompt_caching.py:243: UserWarning: AnthropicPromptCachingMiddleware caching middleware only supports Anthropic models, not instances of <class 'tests.unit_tests.middleware.test_prompt_caching.FakeToolCallingModel'>
    result = await middleware.awrap_model_call(fake_request, mock_handler)

tests/unit_tests/test_chat_models.py::test_anthropic_fields_in_model_kwargs
  /nix/store/5bhzpq0n9rfpnxh4ni91cn4rw1wwprvd-python3.13-pluggy-1.6.0/lib/python3.13/site-packages/pluggy/_callers.py:121: UserWarning: Parameters {'max_tokens_to_sample'} should be specified explicitly. Instead they were passed in as part of `model_kwargs` parameter.
    res = hook_impl.function(*args)

tests/unit_tests/test_chat_models.py::test_anthropic_fields_in_model_kwargs
  /nix/store/5bhzpq0n9rfpnxh4ni91cn4rw1wwprvd-python3.13-pluggy-1.6.0/lib/python3.13/site-packages/pluggy/_callers.py:121: UserWarning: Parameters {'max_tokens'} should be specified explicitly. Instead they were passed in as part of `model_kwargs` parameter.
    res = hook_impl.function(*args)

tests/unit_tests/test_llms.py::test_anthropic_model_params
  /nix/store/7m1n2rqb316jd4qdyn51m0jcgdpmrbyh-python3.13-pydantic-2.11.7/lib/python3.13/site-packages/pydantic/main.py:253: UserWarning: This Anthropic LLM is deprecated. Please use `from langchain_anthropic import ChatAnthropic` instead
    validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)

tests/unit_tests/test_standard.py::TestAnthropicStandard::test_with_structured_output[PersonA]
  /nix/store/d26gd8dv736q1fvkmcvlp3rb4gws9qnf-python3.13-langchain-tests-1.0.0/lib/python3.13/site-packages/langchain_tests/unit_tests/chat_models.py:1005: UserWarning: Unrecognized structured output method 'json_mode'. Defaulting to 'json_schema' method.
    assert model.with_structured_output(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
--------------------------- snapshot report summary ----------------------------
```

### Description

While running pytest on langchain-anthropic, `TestAnthropicStandard.test_with_structured_output[PersonB]` passes a bad schema to the `anthropic` package, causing the test to fail.

### System Info

n/a

--- a/libs/standard-tests/pyproject.toml
+++ b/libs/standard-tests/pyproject.toml
@@ -9,10 +9,10 @@ license = { text = "MIT" }
 readme = "README.md"
 authors = []
 
-version = "1.0.1"
+version = "1.0.2"
 requires-python = ">=3.10.0,<4.0.0"
 dependencies = [
-    "langchain-core>=1.0.3,<2.0.0",
+    "langchain-core>=1.1.0,<2.0.0",
     "pytest>=7.0.0,<9.0.0",
     "pytest-asyncio>=0.20.0,<2.0.0",
     "httpx>=0.28.1,<1.0.0",

